thavalon-server/src/game/builder.rs

```diff
@@ -42,15 +46,16 @@ impl GameBuilder {
     /// Start the game. This consumes `self` because no new players can be added once the game starts.
     /// The returned [`task::JoinHandle`] will complete once the game has ended. The [`Snapshots`] may be
     /// used to track per-player snapshots of the game state.
-    pub fn start(self) -> Result<(Snapshots, task::JoinHandle<()>), CreateGameError> {
+    pub fn start(self, mut lobby_channel: LobbyChannel, abort_registration: AbortRegistration) -> Result<(Snapshots, task::JoinHandle<std::result::Result<(), futures::future::Aborted>>), CreateGameError> {
```

Comment:
The `JoinHandle` that `start` returns can also be used to abort the game task (it has an `abort` method), so I think we should use that instead of an additional `Abortable` wrapper

Reply:
from docs it looks like that was added in tokio 0.3, but we're on 0.2 and migration looks non-trivial. It's probably easier to handle that change separately.

Reply:
Right, forgot we're on an old version of Tokio :cry: